### PR TITLE
PPLUS-510: Automatically apply code changes (with integrator) during …

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/PPLUS-510

Changelog: 
* Integrator adapted for works as part of SDK

### CODE REVIEW

File: src/ManifestStrategy/ConfigureModuleManifestStrategy.php [hunk 1]
```diff
@@ -61,6 +61,16 @@ public function apply(array $manifest, string $moduleName, InputOutputInterface
                 );
             }
 
+            if (!is_bool($value) && !$value) {
```
Comment (alexanderM91) [inline src/ManifestStrategy/ConfigureModuleManifestStrategy.php:64]: Hi @bohdan-turchyk-spryker, do we have a separate bug ticket for this case?

Comment (bohdan-turchyk-spryker) [reply src/ManifestStrategy/ConfigureModuleManifestStrategy.php:64]: https://spryker.atlassian.net/browse/PPLUS-1303 contains links to problematic manifests

**Review by alexanderM91 — APPROVED** [review]

**Review by gechetspr — CHANGES_REQUESTED** [review]
One small mandatory thing to fix and a question

File: composer.json [hunk 2]
```diff
@@ -1,14 +1,13 @@
 {
     "name": "spryker-sdk/integrator",
     "bin": ["bin/integrator"],
-    "type": "composer-plugin",
     "description": "Integrator module",
     "license": "proprietary",
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
```
[the remark this answers is not in the document]
Comment (alexanderM91) [reply composer.json:7]: In this case let's have a develop branch as source of truth.
The main problem that SprykerSDK required PHP 8.0.

[the remark this answers is not in the document]
Comment (alexanderM91) [reply composer.json:7]: Also, in the next release PHP 8.0 will be the default one.

**Review by gechetspr — APPROVED** [review]

File: tests/_data/manifests/src/integrator-manifests-master/Spryker/TestIntegratorConfigureModule/1.0.3/installer-manifest.json [hunk 3]
```diff
@@ -156,6 +156,12 @@
             {
                 "static::IS_CART_CART_ITEMS_VIA_AJAX_LOAD_ENABLED_CHANGED": true
             }
+        },
+        {
+            "target": "\\Pyz\\Zed\\TestIntegratorDefault\\TestIntegratorDefaultConfig::BOOL_VALUE_MISSING"
```
[the remark this answers is not in the document]
Comment (alexanderM91) [reply tests/_data/manifests/src/integrator-manifests-master/Spryker/TestIntegratorConfigureModule/1.0.3/installer-manifest.json:161]: Let's create a follow up ticket for this as it's quite a lot test with the project namespaces

[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply tests/_data/manifests/src/integrator-manifests-master/Spryker/TestIntegratorConfigureModule/1.0.3/installer-manifest.json:161]: Created dedicated issue https://spryker.atlassian.net/browse/PPLUS-1307

**Pull request merged**